cmd/metricsexporter/metricsExporter.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"flag"
+	"net"
+	"net/http"
+	"os"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/prometheus/common/promlog"
+	promflag "github.com/prometheus/common/promlog/flag"
+	log "github.com/sirupsen/logrus"
```

Comment:
do we need logrus package for logging?  Other components in the operator are using `"sigs.k8s.io/controller-runtime/pkg/log"` . We can use the same for consistency.

Reply:
Removed the `logrus` package and added the above suggested (existing) log package